ldpd/ldpd.c

```diff
@@ -277,9 +277,6 @@ main(int argc, char *argv[])
 		exit(1);
 	}
 
-	openzlog(ldpd_di.progname, "LDP", 0,
```

Comment:
Pushed a modified version with an `if (lflag || eflag)` on the `openzlog`, that should work?